todoman/cli.py

```diff
@@ -171,10 +171,12 @@ def cli(ctx, color, porcelain):
 @click.argument('summary', nargs=-1)
 @click.option('--list', '-l', callback=_validate_list_param,
               help='The list to create the task in.')
+@click.option('--location',
+              help='The location of the todo.')
```

Comment:
when I changed in ```_todo_property_option``` that is suggested by you(@untitaker)  my test_edit function in test_ basic.py  failed
```(venv) anubha@anubha-Inspiron-3542:~/Desktop/todoman$ pytest tests/test_basic.py -k test_edit -s
============================================================= test session starts ==============================================================
platform linux -- Python 3.5.2, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /home/anubha/Desktop/todoman, inifile: tox.ini
plugins: hypothesis-3.6.1, cov-2.4.0
collected 28 items 

tests/test_basic.py F

=================================================================== FAILURES ===================================================================
__________________________________________________________________ test_edit ___________________________________________________________________

runner = <click.testing.CliRunner object at 0x7f7146bc7f60>, default_database = <todoman.model.Database object at 0x7f7146bc7710>

    def test_edit(runner, default_database):
        todo = FileTodo()
        todo.list = next(default_database.lists())
        todo.summary = 'Eat paint'
        todo.location = 'Boston'
        todo.due = datetime.datetime(2016, 10, 3)
        todo.save()
    
        result = runner.invoke(cli, ['edit', '1', '--due', '--location', '2017-02-01'])
>       assert not result.exception
E       assert not SystemExit(2,)
E        +  where SystemExit(2,) = <Result SystemExit(2,)>.exception

tests/test_basic.py:422: AssertionError
``` and I also changed in todo_edit function so, I don't understand why it is failed.